scripts/template/generate_transfers.py

```diff
@@ -0,0 +1,103 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Blockchain:
+    name: str
+    native_contract_address: str
+    wrapped_token_deposit: str
+    wrapped_token_withdrawal: str
+
+blockchains = [
+    Blockchain("arbitrum", "'0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'", None, None),
+    Blockchain("avalanche_c", None, None, None),
+    Blockchain("base", None, None, None),
+    Blockchain("celo", None, None, None),
+    Blockchain("ethereum", "'0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'", "source('zeroex_ethereum', 'weth9_evt_deposit')", "source('zeroex_ethereum', 'weth9_evt_withdrawal')"),
+    Blockchain("fantom", None, None, None),
+    Blockchain("gnosis", None, None, None),
+    Blockchain("optimism", None, None, None),
+    Blockchain("polygon", None, "source('mahadao_polygon', 'wmatic_evt_deposit')", "source('mahadao_polygon', 'wmatic_evt_withdrawal')"),
+    Blockchain("zksync", None, None, None),
```

Comment:
Maybe zksync uses this address `0x000000000000000000000000000000000000800a` for the ETH token:
https://explorer.zksync.io/address/0x000000000000000000000000000000000000800a

```suggestion
    Blockchain("zksync", "'0x000000000000000000000000000000000000800a'", None, None),
```